.github/plugins-filter.yml

```diff
@@ -0,0 +1,81 @@
+tauri-plugin-app:
+  - .github/workflows/lint-rust.yml
```

Comment:
note that this file is specific to `lint-rust.yml` only and should be changed to `test-rust.yml` for the test workflow.

Reply:
Ahh let's keep it duplicated for now then